scanpy/plotting/_anndata.py

```diff
@@ -1607,7 +1608,8 @@ def correlation_matrix(
     **kwds
         Only if `show_correlation` is True:
         Are passed to :func:`matplotlib.pyplot.pcolormesh` when plotting the
-        correlation heatmap. Useful values to pas are `vmax`, `vmin` and `cmap`.
+        correlation heatmap. Useful values to pass are `vmax`, `vmin`,
+        `vcenter` and `cmap`.
```

Comment:
Since we're doing special handling of these values, can they be explicit keyword arguments? I don't really like modifying the `**kwargs` dict. Also, `vcenter` isn't an argument to `pcolormesh`, right?

Reply:
OK, this is done I think, we don't modify kwargs any more. Please have a look @ivirshup